packages/mobile/src/send/SendAmount/SendAmountValue.tsx

```diff
@@ -116,29 +128,27 @@ const styles = StyleSheet.create({
   symbolContainer: {
     justifyContent: 'center',
   },
-  maxButtonContainer: {
-    padding: 8,
-  },
-  maxButton: {
+  button: {
     color: colors.gray4,
   },
-  swapInput: {
-    padding: 8,
-  },
   mainSymbol: {
     ...fontStyles.regular,
     fontSize: 24,
     lineHeight: 64,
-    marginHorizontal: 2,
   },
   secondarySymbol: {
     ...fontStyles.small,
     marginHorizontal: 2,
   },
+  // font family, font weight and width are fixes for truncated text on Android https://github.com/facebook/react-native/issues/15114
   mainAmount: {
     ...fontStyles.regular,
     fontSize: 64,
     lineHeight: undefined,
+    fontFamily: 'arial',
```

Comment:
just checking, is it possible to use the Inter or Jost fonts from `fonts.tsx`? if i understand the workaround for the above github issue correctly, we can still use our app fonts?